### PR TITLE
Add `rel="noreferrer"` where possible and switch to HTTPS

### PR DESCRIPTION
**Pull request by LukasReschke**
Just to follow good practise and prevent some automated scanners to complain about "Cross-domain Referer leakage".

### CODE REVIEW

**Comment by LukasReschke** [comment]
cc @MorrisJobke @th3fallen Easy one :)

**Comment by th3fallen** [comment]
yep those are noreferrers.... :+1:

File: core/templates/layout.base.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 		<?php print_unescaped($_['headers']); ?>
 	</head>
 	<body id="body-public">
-		<noscript><div id="nojavascript"><div><?php print_unescaped($l->t('This application requires JavaScript for correct operation. Please <a href="http://enable-javascript.com/" target="_blank">enable JavaScript</a> and reload the page.')); ?></div></div></noscript>
+		<noscript><div id="nojavascript"><div><?php print_unescaped($l->t('This application requires JavaScript for correct operation. Please <a href="http://enable-javascript.com/" target="_blank" rel="noreferrer">enable JavaScript</a> and reload the page.')); ?></div></div></noscript>
```
Comment (nickvergessen) [inline core/templates/layout.base.php]: when changing the language string, we should propably remove the a-tag from the translation and inject it instead?

Comment (LukasReschke) [inline core/templates/layout.base.php]: Good point. Should make it less pain in the future. Will adjust later.

**Comment by PVince81** [comment]
The footer probably needs one too ?
credits-footer

**Comment by LukasReschke** [comment]
Adjusted as well. For the future we might use http://w3c.github.io/webappsec/specs/referrer-policy/ once approved.

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**